prusa/link/web/files_legacy.py

```diff
@@ -0,0 +1,670 @@
+"""/api/files legacy endpoint handlers
+This is a deprecated legacy code"""
+import logging
+from base64 import decodebytes
+from datetime import datetime
+from functools import wraps
+from hashlib import md5
+from io import FileIO
+from os import makedirs, replace, statvfs, unlink
+from os.path import abspath, basename, dirname, exists, getctime, getsize, \
+    join, isdir
+from shutil import move, rmtree
+from time import sleep, time
+
+from poorwsgi import state
+from poorwsgi.request import FieldStorage
+from poorwsgi.response import FileResponse, JSONResponse, Response
+from poorwsgi.results import hbytes
+from prusa.connect.printer import const
+from prusa.connect.printer.const import Source
+from prusa.connect.printer.download import (Transfer, TransferRunningError,
+                                            filename_too_long,
+                                            foldername_too_long,
+                                            forbidden_characters)
+from prusa.connect.printer.metadata import FDMMetaData, get_metadata
+
+from .. import conditions
+from ..const import LOCAL_STORAGE_NAME, PATH_WAIT_TIMEOUT, \
+    HEADER_DATETIME_FORMAT
+from ..printer_adapter.command_handlers import StartPrint
+from ..printer_adapter.job import Job, JobState
+from ..printer_adapter.prusa_link import TransferCallbackState
+from .lib.auth import check_api_digest
+from .lib.core import app
+from .lib.files import (file_to_api, gcode_analysis, get_os_path, local_refs,
+                        sdcard_refs, sort_files)
+
+log = logging.getLogger(__name__)
+
+
+def check_filename(filename):
```

Comment:
Are you sure, that this functions will be used only in this legacy file? If no, please add (leave) them in web/files.py file.

Reply:
I didn't delete them